### ISSUE TITLE
DelugeVPN no .ovpn file log error crit

### ISSUE THREAD

**Issue by gree05155** [binhex/arch-delugevpn#137]
Hello, 
I have a issue with DelugeVPN it says it is running however I can't access it.  I see in the logs it can't find the .ovpn file inside /config/openvpn however whenever I check the contents of my openvpn I see this output:

`user@L303:~$ 
user@L303:~$ ls /config/openvpn/
Toronto.ovpn`

I have attempted a complete reinstall, added removed reconfigured everything. 

I looked at #https://github.com/GhostWriters/DockSTARTer/issues/71 however my transmission works fine.  I can access it no problem, but it doesn't complete the torrents the files are never found it take the torrent downloads it then the file disappears. 

Not sure what else to include.  

Thanks

**Log files**

`
2019-10-28 08:49:05.198084 [info] System information Linux L303 5.0.0-32-generic #34~18.04.2-Ubuntu SMP Thu Oct 10 10:36:02 UTC 2019 x86_64 GNU/Linux

2019-10-28 08:49:05.291549 [info] PUID defined as '1000'

2019-10-28 08:49:05.456522 [info] PGID defined as '1000'

2019-10-28 08:49:05.827698 [warn] UMASK not defined (via -e UMASK), defaulting to '000'

2019-10-28 08:49:05.868975 [info] Permissions already set for volume mappings

2019-10-28 08:49:05.937984 [info] DELUGE_DAEMON_LOG_LEVEL not defined,(via -e DELUGE_DAEMON_LOG_LEVEL), defaulting to 'info'

2019-10-28 08:49:05.978712 [info] DELUGE_WEB_LOG_LEVEL not defined,(via -e DELUGE_WEB_LOG_LEVEL), defaulting to 'info'

2019-10-28 08:49:06.025613 [info] VPN_ENABLED defined as 'yes'

2019-10-28 08:49:06.142157 [crit] No OpenVPN config file located in /config/openvpn/ (ovpn extension), please download from your VPN provider and then restart this container, exiting...`

**Comment by binhex** [comment]
this looks a bit odd, whats with the spaces at the front of the filename?:-

```
user@L303:~$ user@L303:~$ ls /config/openvpn/ Toronto.ovpn
```

can you rename the file and just make sure it doesnt have any leading or trailing spaces on the filename.

**Comment by gree05155** [comment]
I apologize, the file is /config/openvpn/Toronto.ovpn the way I copied and pasted it made it look like it was /conf/openvpn/ Toronto.ovpn , the file has no leading spaces and no spaces at the end. I looked inside Toronto.ovpn and saw that the contents were correct.  Grabbed it from [https://www.privateinternetaccess.com/openvpn/openvpn.zip](url) but also created my own openvpn file from PIA while logged in just in case both didn't work. Wanted to also note, that I have tried [https://github.com/binhex/arch-delugevpn](url) and it gave me the same issue I don't understand why.

**Comment by binhex** [comment]
can you paste your docker run or docker compose file

**Comment by gree05155** [comment]
Sure thing,

When I was using Arch-delugevpn this would be what my command would look like this:

`docker run -d \
    --cap-add=NET_ADMIN \
    -p 8112:8112 \
    -p 8118:8118 \
    -p 58846:58846 \
    -p 58946:58946 \
    --name=<container name> \
    -v /root/docker/data:/data \
    -v /root/docker/config:/config \
    -v /etc/localtime:/etc/localtime:ro \
    -e VPN_ENABLED=yes \
    -e VPN_USER=****** \
    -e VPN_PASS=****** \
    -e VPN_PROV=pia \
    -e STRICT_PORT_FORWARD=yes \
    -e ENABLE_PRIVOXY=yes \
    -e LAN_NETWORK=192.168.2.0/24 \
    -e NAME_SERVERS=209.222.18.222,84.200.69.80,37.235.1.174,1.1.1.1,209.222.18.218,37.235.1.177,84.200.70.40,1.0.0.1 \
    -e DELUGE_WEB_LOG_LEVEL=info\
    -e DEBUG=false \
    -e UMASK=000 \
    -e PUID=1000 \
    -e PGID=1000 \
    binhex/arch-delugevpn`

And inside portanier.io this is what I would see however, portanier.io isn't using Arch-delugevpn  it is using Dockstarter, kept majority of it default and put in what I needed to put in.  Dockstarter got the container to work but still no web-ui however, Arch-delugevpn only got as far as starting then exiting with a code 0 because of the missing open vpn file.  

![InkedDocker1_LI](https://user-images.githubusercontent.com/37492387/67703894-8d200600-f98a-11e9-9bfb-86f31a0bd5d2.jpg)
![Docker2](https://user-images.githubusercontent.com/37492387/67703896-8d200600-f98a-11e9-8223-5c0aeedff8f9.PNG)
![Docker](https://user-images.githubusercontent.com/37492387/67703898-8d200600-f98a-11e9-8b06-512fa95b5a57.PNG)
![Docker3](https://user-images.githubusercontent.com/37492387/67703900-8db89c80-f98a-11e9-8d44-1ed66294cc81.PNG)



`

**Comment by binhex** [comment]
can you remove that volume mapping for /config/openvpn/ that shouldnt be there and maybe causing issues, then copy your ovpn file to host path:- ```/home/user/.config/appdata/delugevpn/openvpn/```

**Comment by gree05155** [comment]
Yes! That worked removing the volume and coping the file directly to /home/user/.config/appdata/delugevpn/openvpn/Toronto.ovpn got me to the webUI. Thanks for assisting me with this.  Not sure if this is a issue for others but now you have a reference. Thanks! 

However, I still need help with my torrents completed files.  I put in "Add Torrents" a torrent file and under the tab Options, I see the download folder by default is set to /home/nobody/Downloads.  Even weirder, I changed the download folder to /home/user/Downloads and the torrent stopped downloading all together? I don't understand how that is possible... I tried to make the torrent move to a completed folder (/home/user/Downloads) and when the file gets completed it is nowhere to be found, checked Downloads and ran a Find command to see if I could find it anywhere on the disk, no luck.  No sure what to do here.


![Screenshot from 2019-10-30 14-10-32](https://user-images.githubusercontent.com/37492387/67899223-12d4ba80-fb1f-11e9-9f52-024e9f27565d.png)

**Comment by gree05155** [comment]
Solved issue by going into WebUI and changing "Download to:" from /home/nobody/Downloads to /Data and it worked.

**Comment by gree05155** [comment]
Hello, that original solution worked.. for the Virtual Machine I was using then. Then my Virtual Machine where you helped me crashed. I can no longer open it and I can't transfer the Virtual Machine, mostly because I was using Virtual Fusion 8 which was released on November 16, 2017 and was the last version that Supported my 2012 xeon processor.  Now I had to re-do all of inside a virtual box VM.  Everything works fine Deluge WebUI works grabs a IP tunnel is good but I can't download anything.  When I add a torrent the download speed goes down and won't download at all.  I would assume this would be because the folder isn't accessible to the container? I tried to rename remove readd the volumes made sure my shared folder between my Mac (host) and Ubuntu (guest) was working.  The original way I did it was just changed it from /home/nobody/Downloads to /Data but now that doesn't work and I don't understand why.
